code/modules/power/generators/turbine.dm

```diff
@@ -204,7 +256,7 @@
 // rpms, TurbGenG modifies the shape of the curve - the lower the value the less straight the curve is.
 
 #define TURBPRES 9000000
```

Comment:
I cant see anywhere in our current codebase, or this PR where this define is used
```suggestion
```